org/issue.ts

```diff
@@ -4,27 +4,89 @@ const gh = danger.github as any;
 const issue = gh.issue;
 const repo = gh.repository;
 
-const qWords: string[] = ["who", "what", "where", "when", "why", "how"]
+const title = issue.title.toLowerCase()
+
+var labelsToAdd: string[] = []
+
+// label: question
+
+const qWords: string[] = ["how", "who", "what", "where", "when", "why", "which"]
 
 if (issue.title.slice(-1) == "?") {
-  addLabel()
+  labelsToAdd.push("question")
 } else {
```

Comment:
What about putting this `else` block in a separate function (`containsQuestionWord`?).
Maybe the `if` too.

Then you can:

```ts
if endsWithQuestionMark() || containsQuestionWord() {
```

Reply:
Thanks for the feedback! I'll try and make it look prettier tomorrow 😝